pkg/azuredisk/azuredisk.go

```diff
@@ -57,6 +57,9 @@ type DriverOptions struct {
 	UseCSIProxyGAInterface     bool
 	EnableDiskOnlineResize     bool
 	AllowEmptyCloudConfig      bool
+	EnableListVolumes          bool
+	EnableListSnapshots        bool
+	RestClientQps              int
```

Comment:
apparently this has to be spelled QPS

Reply:
Yes, you are right. For some reason I thought go was always mixedCaps even for acronyms but apparently those should have a consistent case.